### ISSUE TITLE
[Worker] - Test GHM Whitelisted Org

### ISSUE THREAD

**Issue by adrian-codecov** [codecov/engineering-team#1106]
Ticket to test this https://github.com/codecov/engineering-team/issues/935

**Comment by adrian-codecov** [comment]
Expected behaviors when in GHM plan (users):

- Private repo 
  - Existing PR comment for non-activated member - not activated message ✅ 
  - Existing PR comment for activated member - coverage data
  - New PR comment for non-activated member - not activated message ✅
  - New PR comment for activated member - 
- Public repo
  - Existing PR comment for non-activated member - not activated message 
  - Existing PR comment for activated member - coverage data
  - New PR comment for non-activated member - not activated message ✅
  - New PR comment for activated member - coverage data

Not activate message
![image (1)](https://github.com/codecov/engineering-team/assets/82913673/76b34704-1bc9-47e1-93a1-94a4d0d94cd9)

**Comment by adrian-codecov** [comment]
This ticket is ready to be sent to production, pending on future instructions
